runners/spark/src/main/java/org/apache/beam/runners/spark/SparkPipelineRunner.java

```diff
@@ -115,6 +120,21 @@ public static SparkPipelineRunner fromOptions(PipelineOptions options) {
   }
 
   /**
+   * Creates and returns a new SparkPipelineRunner with specified options.
+   * The Pipeline will be run with the provided Java Spark Context.
+   *
+   * @param options The PipelineOptions to use when executing the job.
+   * @param pjsc a provided Spark Java context to use when executing the job.
+   * @return A pipeline runner that will execute with specified options.
+   */
+  public static SparkPipelineRunner fromOptions(PipelineOptions options, JavaSparkContext pjsc) {
```

Comment:
idem here, I don't think having a second fromOptions method makes sense for such a puntual case, if you pass the info in the options you can infer in the original fromOptions if you have or no the JavaSparkContext and use it if this is the case.